randovania/games/dread/logic_database/Cataris.txt

```diff
@@ -2814,7 +2814,12 @@ Extra - asset_id: collision_camera_040
   > Dock to Thermal Device Room North
       Can Slide
   > Event - Push Wide Beam Block
-      Wave Beam and Can Slide and Push Wide Beam Block
+      All of the following:
+          Wave Beam and Can Slide
```

Comment:
Wave + Charge doesn't actually need the slide. You would only need Slide if you are using the melee charge shot.

Reply:
Do you not need a slide to get into that area of the room?

Reply:
![image](https://github.com/randovania/randovania/assets/46821362/d33603be-cd4e-4a17-aa5c-38a2ad440523)

Reply:
wave + charge you can just shoot from directly below the block, whereas without charge you'll need to slide in to melee the enemy.

Reply:
updated